src/Strategy/Go/Gostd.hs

```diff
@@ -0,0 +1,87 @@
+module Strategy.Go.Gostd (
+  GoStdlibDep (..),
+  listGoStdlibPackages,
+  filterGoStdlibPackages,
+  parseGoStdlibPackages,
+  parseGoStdlibPackage,
+) where
+
+import Control.Algebra (Has)
+import Control.Effect.Diagnostics (Diagnostics)
+import Data.Char (isSpace)
+import Data.Set (Set)
+import Data.Set qualified as Set
+import Data.String.Conversion (toText)
+import Data.Text (Text)
+import Data.Void (Void)
+import DepTypes (DepType (GoType), Dependency (dependencyName))
+import Effect.Exec (AllowErr (Never), Command (..), Exec, execParser)
+import Graphing (Graphing, filter)
+import Path (Abs, Dir, Path)
+import Text.Megaparsec (MonadParsec (eof), Parsec, empty, many, takeWhile1P)
+import Text.Megaparsec.Char (space1)
+import Text.Megaparsec.Char.Lexer qualified as L
+
+type Parser = Parsec Void Text
+
+data GoStdlibDep = GoStdlibDep
+  { goStdlibDepType :: DepType
+  , goStdlibDepName :: Text
+  }
+  deriving (Show, Eq)
+
+mkDep :: Text -> GoStdlibDep
+mkDep = GoStdlibDep GoType
+
+goListStdCommand :: Command
+goListStdCommand =
+  Command
+    { cmdName = "go"
+    , cmdArgs = ["list", "std"]
+    , cmdAllowErr = Never
+    }
+
+listGoStdlibPackages :: (Has Exec sig m, Has Diagnostics sig m) => Path Abs Dir -> m [GoStdlibDep]
+listGoStdlibPackages root = execParser parseGoStdlibPackages root goListStdCommand
+
+parseGoStdlibPackages :: Parser [GoStdlibDep]
+parseGoStdlibPackages = many parseGoStdlibPackage <* eof
```

Comment:
Do you not need a separator here?  Maybe I don't know `Megaparsec` well enough, but I don't see any place that you are allowing any separator between them.  Just lots of `ident` back to back.